recipes/r-rjson/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
You can open a PR at conda/conda-build. For now, I use a cleanup script that simply copies over minimal build scripts to each recipe I have created. I will post it here #2009.